release-notes/1.20.0.md

```diff
@@ -0,0 +1,50 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.20.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* Introduced support for creating virtual machine instances for use cases that require a stronger isolation. With the 1.20 version, you can use both containers and virtual machines for your applications.<!--AC-1459-->
+* Implemented lazy image download to download images only when necessary. AMS will download the image from the image server at first use. <!--AC-1158-->
```

Comment:
Lets call out alpha support for Vulkan graphics

`* Alpha quality support for Vulkan on NVIDIA GPUs is now available. Enabling requires newer NVIDIA drivers (>= 545) and is kept behind a feature flag. Bugs and crashes are known and expected at this point. Stable production support is expected for the 1.21 release of Anbox Cloud. If you are interested in early testing, please reach out.`

Reply:
`* The required NVIDIA driver version has been updated from 525 to 535.`

Reply:
Thanks! I have gotten everything in now except AC-1923, I will add that after a discussion

Reply:
Done for AC-1923 as well